schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/ActivityCreator.java

```diff
@@ -17,6 +21,7 @@
 public class ActivityCreator {
 
     public static final int MaximumRepeatCounter = 25; // Maximum number of activities a repeat will create
+    private static final DateTimeZone ZurichZone = DateTimeZone.forID(InternationalizationUtil.ZURICH_TIME_ZONE_ID);
```

Comment:
Please move this to `com.jasify.schedule.appengine.util.InternationalizationUtil`

I would like to centralize i18n stuff there so that it is easier later when we want to support other Time Zones for example